tests/test_mod.py

```diff
@@ -11,6 +11,23 @@
 from sentinelsat.sentinel import (InvalidChecksumError, SentinelAPI, SentinelAPIError, convert_timestamp, format_date,
                                   get_coordinates, md5_compare)
 
+_small_query = dict(
+        area='0 0,1 1,0 1,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 1, 2))
+
+_large_query = dict(
+        area='0 0,0 10,10 10,10 0,0 0',
+        initial_date=datetime(2015, 1, 1),
+        end_date=datetime(2015, 12, 31))
+
+_api_auth = dict(
```

Comment:
Actually just `environ.get('USERNAME')`. The scihub integration tests are [expected to fail](https://github.com/ibamacsr/sentinelsat/blob/0ed829b211811cf16898acd7c6726bfb853b24cd/.travis.yml#L17) on PRs from forks.

With Travis there is no way to securely provide credentials for the integration tests unless the PR comes from someone with write access to the repository.